content/applications/sales/sales/invoicing/proforma.rst

```diff
@@ -1,34 +1,54 @@
-========================
-Send a pro-forma invoice
-========================
+==================
+Pro-forma invoices
+==================
+
+A *Pro-Forma Invoice* is an abridged or estimated invoice sent in advance of a delivery of goods.
+It notes the kind and quantity of goods, their value, and other important information, such as
+weight and transportation charges.
 
-A pro-forma invoice is an abridged or estimated invoice in advance of a
-delivery of goods. It notes the kind and quantity of goods, their value,
-and other important information such as weight and transportation
-charges. Pro-forma invoices are commonly used as preliminary invoices
-with a quotation, or for customs purposes in importation. They differ
-from a normal invoice in not being a demand or request for payment.
+Pro-forma invoices are commonly used as preliminary invoices with a quotation. They are also used
+for customs purposes during importation. They differ from a normal invoice in that they are not a
+demand (or request) for payment.
 
 Activate the feature
 ====================
 
-Go to :menuselection:`SALES --> Configuration --> Settings` and activate
-the *Pro-Forma Invoice* feature.
+In order to utilize pro-forma invoices, the :guilabel:`Pro-Forma Invoice` feature **must** be
+activated. To do that, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and
+in the :guilabel:`Quotations & Orders` section, click the checkbox next to :guilabel:`Pro-Forma
+Invoice`. Then, click :guilabel:`Save` to save all changes.
```

Comment:
using italics since the active instruction hasn't begun yet
```suggestion
In order to utilize pro-forma invoices, the *Pro-Forma Invoice* feature **must** be activated. To do
that, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and in the
:guilabel:`Quotations & Orders` section, click the checkbox next to :guilabel:`Pro-Forma Invoice`.
Then, click :guilabel:`Save` to save all changes.
```